modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryUserRelLocalServiceTest.java

```diff
@@ -432,6 +519,38 @@ private AccountEntryUserRel _addAccountEntryUserRel(long accountEntryId)
 			_userInfo.suffixId);
 	}
 
+	private AccountEntryUserRel _addPersonTypeAccountEntryUserRel(
+			long accountEntryId)
+		throws Exception {
+
+		UserInfo userInfo = new UserInfo();
+
+		return _accountEntryUserRelLocalService.
+			addPersonTypeAccountEntryUserRel(
+				accountEntryId, TestPropsValues.getUserId(),
+				userInfo.screenName, userInfo.emailAddress, userInfo.locale,
+				userInfo.firstName, userInfo.middleName, userInfo.lastName,
+				userInfo.prefixId, userInfo.suffixId);
+	}
+
+	private void _assertPersonTypeAccountEntryUser(
+			long[] expectedUserIds, long accountEntryId)
```

Comment:
Since person account will only have 1 account user, we don't need to use an array for the `expectedUserId`.

Reply:
@pei-jung I decided to use an array in this case so I can choose to pass in an empty array for the case where I'm expecting no userIds. Does that make sense?

Reply:
We can also use `-1` to indicate that no user is expected, but I'm fine with using an array. Either way works for me.